configs/tour_mode_choice.csv

```diff
@@ -22,7 +22,7 @@ util_DRIVEALONEPAY_Terminal_time,DRIVEALONEPAY - Terminal time,@2 * walktimeshor
 util_DRIVEALONEPAY_Operating_cost,DRIVEALONEPAY - Operating cost,@ivt_cost_multiplier * df.ivot * costPerMile * (odt_skims['SOVTOLL_DIST'] + dot_skims['SOVTOLL_DIST']),,coef_ivt,,,,,,,,,,,,,,,,,,,
 util_DRIVEALONEPAY_Parking_cost,DRIVEALONEPAY - Parking cost,@ivt_cost_multiplier * df.ivot * df.daily_parking_cost,,coef_ivt,,,,,,,,,,,,,,,,,,,
 util_DRIVEALONEPAY_Bridge_toll,DRIVEALONEPAY - Bridge toll,@ivt_cost_multiplier * df.ivot * (odt_skims['SOVTOLL_BTOLL'] + dot_skims['SOVTOLL_BTOLL']),,coef_ivt,,,,,,,,,,,,,,,,,,,
-util_DRIVEALONEPAY_Value_toll,DRIVEALONEPAY - Value toll,@ivt_cost_multiplier * df.ivot * (odt_skims['SOVTOLL_VTOLL'] + dot_skims['SOVTOLL_VTOLL']),,coef_ivt,,,,,,,,,,,,,,,,,,,
+util_DRIVEALONEPAY_Value_toll,DRIVEALONEPAY - Value toll,@ivt_cost_multiplier * df.ivot * (odt_skims['SOVTOLL_VTOLL'] + dot_skims['SOVTOLL_VTOLL']) + cordon_toll  * df.is_dest_cordon - income_low_100_discount * df.is_income_lower_100,,coef_ivt,,,,,,,,,,,,,,,,,,,
```

Comment:
Should the discount also be multiplied by `df.is_dest_cordon` here? Maybe something like

`cordon_toll * df.is_dest_cordon ( 1 - income_low_100_discount * df.is_income_lower_100 )`
if I'm understanding correctly

Reply:
Or, if `income_low_100_discount` is dollars, something like

`df.is_dest_cordon * (cordon_toll - income_low_100_discount * df.is_income_lower_100)`

Reply:
That's right, thanks for noticing !

I also multipied the cordon toll by '@ivt_cost_multiplier * df.ivot '  like the other part of the toll. I did a similar thing for the trip file as well.

Now looks like this:

util_DRIVEALONEPAY_Value_toll,DRIVEALONEPAY - Value toll,@ivt_cost_multiplier * df.ivot * (odt_skims['SOVTOLL_VTOLL'] + dot_skims['SOVTOLL_VTOLL'] + df.is_dest_cordon * (cordon_toll  - income_low_100_discount * df.is_income_lower_100)),,coef_ivt,,,,,,,,,,,,,,,,,,,